jsk.rosbuild

```diff
@@ -14,8 +14,8 @@ function usage {
 
 function error {
     echo "export ROS_HOME=$ROS_HOME"
-    echo "source $ROS_INSTALLDIR/setup.bash"
-    (which rosrun && rosrun rosunit clean_junit_xml.py; echo "done")
+    echo "source $ROS_INSTALLDIR/devel/setup.bash"
+    (cd $ROS_INSTALLDIR; catkin_test_results --all; echo "done")
```

Comment:
once you run catkin catkin, CATKIN_TEST_RESULTS_DIR is set and you do not have to set arguments